src/main/java/seedu/address/logic/parser/IssueCliSyntax.java

```diff
@@ -0,0 +1,14 @@
+package seedu.address.logic.parser;
+
+/**
+ * Contains Command Line Interface (CLI) syntax definitions common to multiple commands
+ */
+public class IssueCliSyntax {
+
+    /* Prefix definitions */
+    public static final Prefix PREFIX_DESCRIPTION = new Prefix("dn/");
+    public static final Prefix PREFIX_DEADLINE = new Prefix("dl/");
+    public static final Prefix PREFIX_STATUS = new Prefix("s/");
+    public static final Prefix PREFIX_PRIORITY = new Prefix("py/");
+    public static final Prefix PREFIX_PROJECTID = new Prefix("pr/");
```

Comment:
Consider refining the prefixes in the next iteration by using more intuitive prefix names